packages/bvaughn-architecture-demo/src/utils/testing.tsx

```diff
@@ -194,6 +195,12 @@ export function createMockReplayClient() {
     loadRegion: jest.fn().mockImplementation(async () => {}),
     removeEventListener: jest.fn(),
     runAnalysis: jest.fn().mockImplementation(async () => []),
-    searchSources: jest.fn().mockImplementation(async () => []),
+    searchSources: jest.fn().mockImplementation(async () => {}),
+    searchFunctions: jest.fn().mockImplementation(async () => {}),
+    getAnnotationKinds: jest.fn().mockImplementation(async () => []),
+    getResponseBody: jest.fn(),
+    getRequestBody: jest.fn(),
+    findKeyboardEvents: jest.fn().mockImplementation(async () => []),
+    findNavigationEvents: jest.fn().mockImplementation(async () => []),
```

Comment:
Small nit but these were alpha sorted before and I find them easier to read/scan quickly that way.